ember-intl/addon/index.ts

```diff
@@ -1 +1 @@
-export { default as IntlService } from './services/intl';
```

Comment:
Unfortunately this was a breaking change. It was previously possible to import the IntlService and subclass it in userland. With this change this is no longer possible. I will open an issue.

Reply:
Ahhh please disregard. I realized it is still possible to `import IntlService from 'ember-intl/services/intl';`.

Reply:
No problem. Yep, the `index` file only re-exports things for an easy access, so you can continue to import the `intl` service with `import IntlService from 'ember-intl/services/intl';`.

I had meant to export only the type of the `intl` service in the `index` file (at least, for now), so it was my bad for introducing a bug that became a soft breaking change.